--- jupyter_core-4.5.0/jupyter_core/tests/test_command.py.orig	2018-02-07 18:03:52.000000000 +0100
+++ jupyter_core-4.5.0/jupyter_core/tests/test_command.py	2019-09-29 13:02:17.751091248 +0200
@@ -134,6 +134,8 @@
     env = {'PATH': ''}
     if 'SYSTEMROOT' in os.environ:  # Windows http://bugs.python.org/issue20614
         env[str('SYSTEMROOT')] = os.environ['SYSTEMROOT']
+    if 'PYTHONPATH' in os.environ:
+        env[str('PYTHONPATH')] = os.environ['PYTHONPATH']
     if sys.platform == 'win32':
         env[str('PATHEXT')] = '.EXE'
     # This won't work on windows unless
@@ -160,6 +162,8 @@
     env = {'PATH':  str(b)}
     if 'SYSTEMROOT' in os.environ:  # Windows http://bugs.python.org/issue20614
         env[str('SYSTEMROOT')] = os.environ['SYSTEMROOT']
+    if 'PYTHONPATH' in os.environ:
+        env[str('PYTHONPATH')] = os.environ['PYTHONPATH']
     if sys.platform == 'win32':
         env[str('PATHEXT')] = '.EXE'
     out = check_output([sys.executable, str(jupyter), 'witness'], env=env)
